deploy/docker/build.sh

```diff
@@ -1,9 +1,11 @@
 #!/bin/bash
 
+IMAGE=${1-heapster:canary}
```

Comment:
Ah maybe my bash is weak :) I intended for this to be "heapster:canary", but this seems like we're selecting a var with that name?